src/Nokia_LCD.cpp

```diff
@@ -29,6 +29,19 @@ Nokia_LCD::Nokia_LCD(const uint8_t clk_pin, const uint8_t din_pin,
       kDc_pin{dc_pin},
       kCe_pin{ce_pin},
       kRst_pin{rst_pin},
+      kBl_pin{NULL},
```

Comment:
This is essentially the same as initializing it to `0`, which may actually be a valid "Arduino pin" (i.e. `PB0` for the ATtiny85).